cpu/atmega1281/include/cpu_conf.h

```diff
@@ -29,17 +29,22 @@ extern "C" {
  * @name Kernel configuration
  *
  * Since printf seems to get memory allocated by the linker/avr-libc the stack
- * size tested sucessfully even with pretty small stacks.k
+ * size tested sucessfully even with pretty small stacks.
  * @{
  */
 #define THREAD_EXTRA_STACKSIZE_PRINTF    (128)
 
 #ifndef THREAD_STACKSIZE_DEFAULT
-#define THREAD_STACKSIZE_DEFAULT   (256)
+#   define THREAD_STACKSIZE_DEFAULT   (256)
```

Comment:
some ancient compilers require to have the `#` at the beginning of the line [1] (bullet 4)

[1] http://www.delorie.com/gnu/docs/gcc/cpp_48.html